CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog Manager GitHub Action changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+**NOTE: This is an automatically generated file. Do not modify contents of this file manually.**
+
+## [1.0.0] - 2023-04-08
```

Comment:
If you're talking about the already existing version 1.0.0, that's fine. I created that manually because that was the state of the action after my first commit. The updated and latest version section should be updated once the workflow runs.

*Note*: I don't have access over the repository, but the workflow should be granted `Read and write` permissions